src/fin_webapp/app/controllers/market_data_controller.rb

```diff
@@ -0,0 +1,8 @@
+class MarketDataController < ApplicationController
+  def index
+    alpha_vantage_service = AlphaVantageService.new
+    @market_data = alpha_vantage_service.get_news_sentiment(params[:symbol])
```

Comment:
why use it as @market_data I dont think its required to be a global variable , replace without the '@'

Reply:
63cae27 resolved